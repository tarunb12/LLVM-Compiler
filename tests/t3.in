int main() {
    printf("%d * (%d + %d) = %d", 5, 6, 7, 5 * (6 + 7))
}