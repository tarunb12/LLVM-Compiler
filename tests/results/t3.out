Error: File "tests/t3.in": Cannot perform the operation "+" on type int -> int.